tests/openwisp2/settings.py

```diff
@@ -82,10 +82,18 @@
 
 ROOT_URLCONF = 'openwisp2.urls'
 
-ASGI_APPLICATION = 'openwisp_controller.geo.channels.routing.channel_routing'
-CHANNEL_LAYERS = {
-    'default': {'BACKEND': 'channels.layers.InMemoryChannelLayer'},
-}
+ASGI_APPLICATION = 'openwisp2.routing.application'
+if TESTING:
+    CHANNEL_LAYERS = {
+        'default': {'BACKEND': 'channels.layers.InMemoryChannelLayer'},
+    }
```

Comment:
We should update the  [.travis.yml](https://github.com/openwisp/openwisp-controller/blob/master/.travis.yml#L16-L19) and remove the configuration for adding `redis-server` in the build. 

Also, If will be nicer to write it as
```
if not TESTING:
```

this will maintain consistency with the code in the same file: 
https://github.com/openwisp/openwisp-controller/blob/e0aec17cd66f846fbb7bcf094563f51e21599905/tests/openwisp2/settings.py#L140-L145

The latter change is upto you though